To: vim_dev@googlegroups.com
Subject: Patch 7.3.215
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.215 (after 7.3.210)
Problem:    Wrong file names in previous patch. (Toothpik)
Solution:   Include the option changes.
Files:      src/option.c, src/option.h


*** ../mercurial/vim73/src/option.c	2011-05-19 13:40:47.000000000 +0200
--- src/option.c	2011-06-12 19:58:17.000000000 +0200
***************
*** 901,906 ****
--- 901,913 ----
  			    {(char_u *)0L, (char_u *)0L}
  #endif
  			    SCRIPTID_INIT},
+     {"cscoperelative", "csre", P_BOOL|P_VI_DEF|P_VIM,
+ #ifdef FEAT_CSCOPE
+ 			    (char_u *)&p_csre, PV_NONE,
+ #else
+ 			    (char_u *)NULL, PV_NONE,
+ #endif
+ 			    {(char_u *)0L, (char_u *)0L} SCRIPTID_INIT},
      {"cscopetag",   "cst",  P_BOOL|P_VI_DEF|P_VIM,
  #ifdef FEAT_CSCOPE
  			    (char_u *)&p_cst, PV_NONE,
*** ../mercurial/vim73/src/option.h	2011-05-19 13:40:47.000000000 +0200
--- src/option.h	2011-06-12 19:58:49.000000000 +0200
***************
*** 391,396 ****
--- 391,397 ----
  EXTERN char_u	*p_cpo;		/* 'cpoptions' */
  #ifdef FEAT_CSCOPE
  EXTERN char_u	*p_csprg;	/* 'cscopeprg' */
+ EXTERN int	p_csre;		/* 'cscoperelative' */
  # ifdef FEAT_QUICKFIX
  EXTERN char_u	*p_csqf;	/* 'cscopequickfix' */
  #  define	CSQF_CMDS   "sgdctefi"
*** ../vim-7.3.214/src/version.c	2011-06-12 22:03:15.000000000 +0200
--- src/version.c	2011-06-12 22:12:34.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     215,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
171. You invent another person and chat with yourself in empty chat rooms.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
